docs/chainlink-keepers/supported-networks.html

```diff
@@ -31,7 +31,11 @@
   - Mainnet
   - Testnet
 - Avalanche:
+  - Mainnet
```

Comment:
Add Avalanche to the home page for Keepers: https://github.com/smartcontractkit/documentation/blob/c7e1e1aee48992eeb8d890dc4c9295fd30d66d17/docs/home.html#L273